app/assets/javascripts/browsered/helpers.js

```diff
@@ -22,7 +22,73 @@ const initialiseAddressCountryAutocomplete = () => {
   document.getElementsByTagName('head')[0].appendChild(autocompleteScript)
 }
 
+const toggleButton = (button) => {
+  if (button) {
+    button[button.getAttribute('disabled') ? 'removeAttribute' : 'setAttribute']('disabled', 'disabled')
```

Comment:
Just wondering what the square brackets do here?

Reply:
It makes it into either `button['removeAttribute']('disabled')` or `button['setAttribute']('disabled', 'disabled'`, which apparently works (this is copied from existing code).

Reply:
Ah - makes sense now.  Clever stuff.